src/web/mjs/connectors/TopToon.mjs

```diff
@@ -0,0 +1,48 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/manga.mjs';
+
+export default class TopToon extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'toptoon';
+        super.label = 'TOPTOON (탑툰)';
+        this.tags = [ 'webtoon', 'korean' ];
+        this.url = 'https://toptoon.com';
+    }
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, 'div.ver02 .tit_toon');
+        const id = uri.pathname + uri.search;
+        const title = data[0].textContent.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        const request = new Request('https://fastidious.toptoon.com/production/comicTotal/504656797d23c25a58c9e6e4890ffeefb0e930abbb449df40ca641736117818e.json', this.requestOptions);
```

Comment:
i could grab the link from the site itself (fetchREGEX or fetchUI), idk if it changes ever

Reply:
It will probably stop working when the website backend is updated

Reply:
even after the link changes it seems the old one still works, its just missing some stuff